EmployeeList/Models/Employee.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Employee.swift
+//  EmployeeList
+//
+//  Created by Wang Siman on 1/29/21.
+//  Copyright © 2021 None. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+enum EmployeeType: String, Decodable {
+    case FULL_TIME
+    case PART_TIME
+    case CONTRACTOR
+}
+
+struct Employee: Decodable, Identifiable {
```

Comment:
I would make this plain Codable, not just Decodable, unless there's a compelling reason not to.

Reply:
This is a very good point! I wrote this very first app by referring to some sample projects on github. For some reason, a similar model in the sample conforms to Decodable. Besides, in terms of current required features, uploading employee to a remote server isn't required. So I thought Decodable is ok...... But since I did use CodingKey to make Employee conform to Decodable, changing Decodable to Codable is fairly easy.